.babelrc

```diff
@@ -1,5 +1,13 @@
 {
-    "presets": ["@babel/preset-env", "@babel/preset-react"],
-    "plugins": ["transform-class-properties"]
-
+  "plugins" : ["transform-class-properties"],
+    "presets": [
+      [
+        "@babel/preset-env", {
+          "targets": {
+            "node": "current"
+          }
+        }
+      ], 
+      "@babel/preset-react"
+    ]
   }
```

Comment:
Then please add a newline to this file l am sure eslint complains about this as well.

Reply:
@jennizalwango this being a file with just a json object. it doesn't need a new line after and **No** the linter is not complaining about it.

Reply:
@jennizalwango kindly note that I implemented your feedback expect using history.go() instead of history.push() because they were both malfunctioning. Kindly check out the review app and review. thanks

Reply:
okay thanks